apps/prairielearn/src/pages/partials/navbar.ejs

```diff
@@ -101,9 +101,9 @@
 
           <% if (locals.authn_is_administrator) { %>
 
-            <a class="dropdown-item" id="navbar-administrator-toggle">
+            <button type="button" class="dropdown-item" id="navbar-administrator-toggle">
```

Comment:
Using `<a>` without an `href` is an anti-pattern. The new markup is more accessible and ensures that a pointer cursor is used when hovering.